docs/portal_config.md

```diff
@@ -160,6 +160,7 @@ Below is an example, with inline comments describing what each JSON block config
     "explorer": true, // required; indicates the flag and whether to hide it or not
     "explorerPublic": true // optional; If set to true, the data explorer page would be treated as a public component and can be accessed without login. Data explorer page would be public accessible if 1. tiered access level is set to libre OR 2. this explorerPublic flag is set to true.
     "discovery": true, // optional; whether to enable the Discovery page. If true, `discoveryConfig` must be present as well.
+    "explorerHideEmpty": false, // optional, when filtering data hide:  FilterSection when they are empty.
```

Comment:
I would prefer this field be added to `explorerConfig.filters`, so each explorer tabs can control their explorer filers individually

And rename it to `hideEmptyFilterSection` if it got moved

Reply:
was going to tie this into other features, but probably best to separate it (so i will rename), I don't think it would be good UI to have items hidden in one tab but not another so im going to leave it separate from the individual tabs for now

Reply:
if you want to keep it in here, then I suggest to rename it to `explorerHideEmptyFilterSection` so it is clear that this is a explorer feature flag

Reply:
updated, thanks for the suggestion